src/components/Webhooks/WebhooksManagement.js

```diff
@@ -140,39 +131,23 @@ class WebhooksManagement extends React.Component{
                 method: 'post',
                 url: 'https://api.badgr.io/o/token',
                 data
-            }).then(res => {
-                console.log(res, res.data.access_token)                
+            }).then(res => {             
```

Comment:
Let's try to avoid using the promise callbacks. Prefer the async and await syntax.